js/news.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+class fetchdata {
+  apiFetcher(url) {
+    fetch(url)
+      .then(response => response.json())
+      .then(myJson => newsSource.articlesProvider(myJson))
+      .catch(error => newsSource.articlesProvider(myJson));
+  }
+  apiSourceFetcher(url) {
+    fetch(url)
+      .then(response => response.json())
+      .then(myJson => newsSource.newsSourceChannel(myJson.sources))
+      .catch(error =>
+        console.log("News Source Channel || Something went wrong on server")
+      );
+  }
+}
+
+class newsSourceProvider extends fetchdata {
+  constructor(data) {
+    super(data);
+  }
+  articlesProvider(data) {
```

Comment:
this is a class method - please name class methods like functions, e.g. `renderArticles` (`<action><description>` - something like that)